packages/kernel-5.15/config-bottlerocket

```diff
@@ -91,6 +109,21 @@ CONFIG_MODULE_COMPRESS_XZ=y
 CONFIG_SERIO_I8042=m
 CONFIG_KEYBOARD_ATKBD=m
 CONFIG_MOUSE_PS2=m
```

Comment:
As I understand it, this config option originally was `=y` in the Amazon Linux kernel and now has been unset. @bcressey changed this to a module to speed up the boot when this was `=y`. We could potentially drop the code entirely now.